tmt/steps/provision/local.py

```diff
@@ -183,6 +187,8 @@ def go(self, *, logger: Optional[tmt.log.Logger] = None) -> None:
 
         data.show(verbose=self.verbosity_level, logger=self._logger)
 
+        self.assert_feeling_safe("1.38", "Local provision plugin")
```

Comment:
The return value must be consumed: `assert_feeling_safe` will report `--feeling-safe` is required, but it will not stop. Maybe the helper could raise an exception instead of returning true or false - I'm not a happy user of exceptions, my tendency is to avoid them, but not checking the return value is an easy and wrong thing to do, and exception would prevent that.

Reply:
ah, I assumed that if provisioning step fails, it won't continue. Will address tomorrow.

Reply:
True, but the step does not fail, only reports a fail, via `fail()`. Step reports a problem and happily continues.

Reply:
Well, can't blame me for thinking `self.fail()` means fail.

Reply:
> Maybe the helper could raise an exception

Raising ProvisioningError